### PR TITLE
Fix typos for `username_available` admin API

### PR DESCRIPTION
**Pull request by binarymaster**
- Fix typo in the word `available`
- HTTP method should be `GET`, not `POST`

### CODE REVIEW

**Review by clokep — APPROVED** [review]
Looks great, thank you so much!

File: changelog.d/11286.doc [hunk 1]
```diff
@@ -0,0 +1 @@
+Fix typo in the word `available` and fix HTTP method (should be `GET`) for the `username_available` admin API.
```
Comment (clokep) [inline changelog.d/11286.doc:1]: We like to credit our contributors, you can include your nick or name if you'd like!

```suggestion
Fix typo in the word `available` and fix HTTP method (should be `GET`) for the `username_available` admin API. Contributed by Stanislav Motylkov. 
```

Comment (binarymaster) [reply changelog.d/11286.doc]: Well, it technically should land in the git history, but ok, added it :slightly_smiling_face:

Comment (clokep) [reply changelog.d/11286.doc]: Yep, but this is what we use to generate the changelogs, which are much more public!

**Pull request merged**